build/build-dt-report-resources.js

```diff
@@ -23,8 +23,21 @@ const htmlReportAssets = require('../lighthouse-core/report/html/html-report-ass
  */
 function convertToAsciiAndWriteFile(name, content) {
   assert(content);
-  // eslint-disable-next-line no-control-regex
-  const escaped = content.replace(/[^\x00-\x7F]/g, c => '\\\\u' + c.charCodeAt(0).toString(16));
+
+  /** @type {string} */
```

Comment:
if going to declare this as a `string`, maybe give it `\\\\u` as the default (instead of doing it in the final `else`) and then override it if html or css

Reply:
ok